R/tm_t_events.R

```diff
@@ -1,22 +1,15 @@
 #' Template: Events by Term
 #'
+#' Creates a valid expression to generate a table of events by term.
+#'
 #' @inheritParams template_arguments
-#' @param label_hlt (`string`)\cr label of the `hlt` variable from `dataname`. The label will be extracted from the
-#' module.
-#' @param label_llt (`string`)\cr label of the `llt` variable from `dataname`. The label will be extracted from the
-#' module.
-#' @param event_type (`character`)\cr type of event that is summarized (e.g. adverse event, treatment).
-#'   Default is "event".
-#' @param sort_criteria (`character`)\cr how to sort the final table. Default option `freq_desc` sorts
-#'   on column `sort_freq_col` by decreasing number of patients with event. Alternative option `alpha` sorts events
-#'   alphabetically.
 #' @param sort_freq_col (`character`)\cr column to sort by frequency on if `sort_criteria` is set to `freq_desc`.
 #' @param incl_overall_sum (`flag`)\cr  whether two rows which summarize the overall number of adverse events
 #'   should be included at the top of the table.
 #'
 #' @seealso [tm_t_events()]
```

Comment:
@Melkiades I also added in a return value for template functions at Pawel's request [here](https://github.com/insightsengineering/teal.modules.clinical/pull/994/files#diff-f965b02c71a0a200d36f300bbb98545e8fbf80071dc63a4154c0797afa79a95fR82)

```suggestion
#' @inherit template_arguments return
#'
#' @seealso [tm_t_events()]
```